sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineManagedDiskOperationsTests.java

```diff
@@ -603,6 +604,7 @@ public void canCreateVirtualMachineWithManagedDiskInManagedAvailabilitySet() {
     }
 
     @Test
+    @DoNotRecord(skipInPlayback = true)
```

Comment:
Do not skip it. Please keep the test. Just you don't need to record it. Use script to change api-version of that recording from main.